src/services/rfqm_service.ts

```diff
@@ -118,8 +118,9 @@ export class RfqmService {
         const marketOperation = isSelling ? MarketOperation.Sell : MarketOperation.Buy;
         const assetFillAmount = isSelling ? sellAmount! : buyAmount!;
 
-        // Prepare gas estimate
+        // Prepare gas estimate and fee
         const gas: BigNumber = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
```

Comment:
What gas estimate is this? `fast`?

Reply:
Correct, it will use the `fast` option from https://ethgasstation.info/api/ethgasAPI.json (and then convert it in terms of gwei)

https://github.com/0xProject/protocol/blob/development/packages/asset-swapper/src/utils/protocol_fee_utils.ts#L68

Reply:
we should call this gasPrice

```suggestion
        const gasPrice: BigNumber = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
```